src/hardware/dreamsourcelab-dslogic/protocol.c

```diff
@@ -874,22 +874,6 @@ static void LIBUSB_CALL receive_transfer(struct libusb_transfer *transfer)
 		resubmit_transfer(transfer);
 }
 
-static int receive_data(int fd, int revents, void *cb_data)
```

Comment:
With this change, drivers no longer need to implement callbacks if all they are doing is calling libusb_handle_events_timeout().  I tried to update a few of them, but frankly IDK if I have the time & energy to update all of them.